lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -0,0 +1,102 @@
+        """
+        Return server configuration information required by the UI
+        dashboard. This includes:
+
+        metadata: Information about the server configuration
+            result_index: The "root" index name for Pbench result data,
+                qualified by the current index version and prefix. In the
+                current ES schema, this is "v5.result-data-sample."
+            result_data_index: The "result-data" index has been broken into
+                "result-data-sample" and "result-data" indices for the
+                Elasticsearch V7 transition. In the current ES schema, this
+                is "v5.result-data."
+            run_index: The "master" run-data index root. In the current ES
+                schema, this is "v6.run-data."
+            run_toc_index: The Elasticsearch V7 index for run TOC data. In
+                the current ES schema, this is "v6.run-toc."
+        api:    A list of the server APIs supported; we give a name, which
+                corresponds to either the current config.json property name
+                (e.g., "elasticsearch") or the current src/service method name
+                being replaced (e.g., "queryControllers")
+            results: Direct access to Pbench data sets for the dashboard; this
+                is currently direct HTTP access through Apache public_html, and
+                not an "API" as such; we will need to adopt this as a real
+                server API when Pbench moves from local filesystem to S3 storage
+                schemas
+            elasticsearch: The Pbench pass-through URI for the Elasticsearch
+                cluster. This will eventually be superceded by the native
+                Pbench APIs, though it might remain accessible with special
+                user/group privileges to support special cases
+            graphql: The GraphQL frontend on postgreSQL currently used by the
+                dashboard user mocks. This will be superceded and deprecated
+                by native Pbench user management APIs
+            queryControllers: Return information about the run documents that
+                were created within a specified range of months.
+            queryMonthIndices: Return the YYYY-mm date strings for all ES
+                vx.run-data.* indices, in descending order
+
+        TODO: We need an internal mechanism to track the active versions of the
+        various Elasticsearch template documents. We're hardcoding them here and
+        in other APIs. We should consider persisting an equivalent of the
+        mapping table built in "indexer.py" for use across the server APIs.
+
+        Meta TODO: We're giving the pass-through APIs for Elasticsearch and
+        GraphQL here, which implies adoption of Nikhil's and Fuqing's work to
+        move the dashboard away from direct access to the backend DB servers.
+        The alternative would be to expose the direct Elasticsearch and GraphQL
+        URIs here.
```

Comment:
Now that https://github.com/distributed-system-analysis/pbench-dashboard/pull/95 has been merged, do we still need this comment?

Reply:
Part of it, until the dashboard drops GraphQL ... but yes, Nikhil's work is now in so that simplifies the comment a bit.

Reply:
Nope; it'll be gone.